.github/workflows/discourse-theme.yml

```diff
@@ -85,6 +91,24 @@ jobs:
         if: ${{ !cancelled() && steps.js-pkg-manager.outputs.manager != env.JS_PKG_MANAGER_NULL_VALUE }}
         run: ${{ steps.js-pkg-manager.outputs.manager }} ember-template-lint --no-error-on-unmatched-pattern javascripts
 
+      - name: Rubocop
+        if: ${{ !cancelled() }}
+        run: | 
+          if test -f .rubocop.yml; then
```

Comment:
I think we could use [hashFiles](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/evaluate-expressions-in-workflows-and-actions#hashfiles) in the `if` check above (it should return empty string when there aren't matching files) instead of this, but this follows how we've set out these checks elsewhere.  @davidtaylorhq

Reply:
I don't mind either way